test/utils/TestKeyManager.sol

```diff
@@ -73,7 +73,7 @@ library TestKeyManager {
     function sign(TestKey memory key, bytes32 hash) internal pure returns (bytes memory) {
         if (key.keyType == KeyType.P256) {
             (bytes32 r, bytes32 s) = vm.signP256(key.privateKey, hash);
-            return abi.encodePacked(r, s);
+            return abi.encode(r, s, false);
```

Comment:
If we wanted to move the signature decoding into calldata too (we would have to port over webauthn decoding), we could do packed here